### PR TITLE
added rectangle

### PR DESCRIPTION
**Pull request by yktemp2**
I have added rectangle as requested by you

### CODE REVIEW

**Comment by yktemp1** [comment]
> I have added rectangle as requested by you

@yktemp2 Can you change the position of rectangle to center it

File: scripts/main.js [hunk 1]
```diff
@@ -5,4 +5,5 @@ function setup(){
 
 function draw(){
     background("#333");
-}
\ No newline at end of file
+    rect(0,0,200,200);
```
Comment (yktemp1) [inline scripts/main.js]: Change Here

**Review by yktemp1 — CHANGES_REQUESTED** [review]

File: scripts/main.js (hunk 1 above)
Comment (yktemp2) [reply scripts/main.js]: Done

**Review by yktemp1 — APPROVED** [review]
This is fine

**Pull request merged**